packages/shared/lib/core/profile/actions/active-profile/logout.ts

```diff
@@ -9,25 +9,27 @@ import { clearPollNetworkInterval } from '@core/network'
 import {
     resetActiveProfile,
     activeProfile,
-    isActiveLedgerProfile,
     isSoftwareProfile,
     activeAccounts,
     lockStronghold,
+    isLedgerProfile,
 } from '@core/profile'
 import { resetSelectedAccount } from '@core/account'
 
 /**
  * Logout from active profile
  */
 export function logout(clearActiveProfile: boolean = false, _lockStronghold: boolean = true): Promise<void> {
-    const { lastActiveAt, loggedIn, hasLoadedAccounts } = get(activeProfile)
+    const { lastActiveAt, loggedIn, hasLoadedAccounts, type } = get(activeProfile)
 
     // (TODO): Figure out why we are using a promise here?
     return new Promise((resolve) => {
         if (_lockStronghold && get(isSoftwareProfile)) {
             lockStronghold()
-        } else if (get(isActiveLedgerProfile)) {
-            stopPollingLedgerStatus()
+        }
+
+        if (isLedgerProfile(type) && isPollingLedgerDeviceStatus) {
```

Comment:
there's no way a profile can be a ledger profile and software profile at the same time right? 
If so, `else if`

Reply:
I can make this change